be/src/main/java/codesquad/issuetracker/service/IssueService.java

```diff
@@ -27,61 +22,46 @@ public class IssueService {
-            countOfIssuesByStatus.getOrDefault(IssueStatus.CLOSED, 0L),
-            issues.stream()
-                .filter(issue -> issue.hasSameStatus(condition.getStatus()))
-                .map(issue -> IssueDto.of(
-                        issue,
-                        issue.getMilestone(),
-                        issue.getAssignees().stream().map(Assignee::getMember)
-                            .collect(Collectors.toList()),
-                        issue.getIssueLabels().stream().map(IssueLabel::getLabel)
-                            .collect(Collectors.toList())
-                    )
-                )
-                .collect(Collectors.toList())
-        );
-    }
+        List<IssueDto> selectedIssues = issues.stream()
+            .filter(issue -> issue.hasSameStatus(condition.getStatus()))
+            .map(IssueDto::of)
+            .collect(Collectors.toList());
 
-    private Set<String> parseConditions(String conditions) {
-        Set<String> exclusionConditions = new HashSet<>();
-        if (conditions == null) {
-            return exclusionConditions;
-        }
+        long[] countOfIssuesByStatus = calculateCountOfIssuesByStatus(issues.size(), selectedIssues.size(), condition.isOpenStatus());
 
-        String[] params = conditions.split(",");
-        for (String param : params) {
-            exclusionConditions.add(param.replaceAll(" ", ""));
-        }
-
-        return exclusionConditions;
-    }
-
-    private Map<IssueStatus, Long> getCountOfIssuesByStatus(List<Issue> issues) {
-        Map<IssueStatus, Long> countOfIssuesByStatus = new HashMap<>();
-        for (Issue issue : issues) {
-            countOfIssuesByStatus.put(issue.getStatus(),
-                countOfIssuesByStatus.getOrDefault(issue.getStatus(), 0L) + 1);
-        }
-        return countOfIssuesByStatus;
+        return new IssueDtos(countOfIssuesByStatus[0], countOfIssuesByStatus[1], selectedIssues);
     }
 
     private boolean containsAllLabels(List<IssueLabel> issueLabels, Set<String> labelConditions) {
-        Set<String> labelsOfIssue = issueLabels.stream()
-            .map(issueLabel -> issueLabel.getLabel().getName()).collect(Collectors.toSet());
+        Set<String> labelsOfIssue = issueLabels
+            .stream()
+            .map(IssueLabel::getLabelName)
+            .collect(Collectors.toSet());
 
         return labelsOfIssue.containsAll(labelConditions);
     }
 
+    private long[] calculateCountOfIssuesByStatus(long totalCount, long countOfSelectedIssues, boolean isOpenStatus) {
+        long countOfOpenIssues, countOfClosedIssues;
+
+        if (isOpenStatus) {
+            countOfOpenIssues = countOfSelectedIssues;
+            countOfClosedIssues = totalCount - countOfSelectedIssues;
+        } else {
+            countOfOpenIssues = totalCount - countOfSelectedIssues;
+            countOfClosedIssues = countOfSelectedIssues;
+        }
+
+        return new long[]{countOfOpenIssues, countOfClosedIssues};
+    }
```

Comment:
반환 타입으로 실제 어레이나 리스트가 아닌 데이터를 어레이,리스트로 반환하는건 좋지 않습니다.
open된 이슈 개수만을 반환하고 해당 정보를 사용 하면 어떨까요

```suggestion
    private long countOpend(long totalCount, long countOfSelectedIssues, boolean isOpenStatus) {
        if (isOpenStatus) {
            return  countOfSelectedIssues;
        } 
        
        return totalCount - countOfSelectedIssues;
    }
```